<launch>
  
	<rosparam file="$(find blueberry-detection-ros)/param/yolov8_configuration.yaml" command="load" />

	<node 
		pkg="blueberry-detection-ros"
		type="detection-ros.py"
		name="detection_node"
		output="screen"
    >
    </node>

</launch>